core/App/components/misc/InfoBox.tsx

```diff
@@ -177,9 +179,12 @@ const InfoBox: React.FC<BifoldErrorProps> = ({
         </Text>
       </View>
       <View style={styles.bodyContainer}>
-        <Text style={styles.bodyText} testID={testIdWithKey('BodyText')}>
-          {showDetails ? message : description}
-        </Text>
+        {bodyContent}
+        {(!!description || !!message) && (
```

Comment:
If `description` or `message` is `undefined` it will be treated as a falsy value we don't really need to explicitly convert it to boolean primitive with `!!`.